packages/@uppy/react/src/Dashboard.d.ts

```diff
@@ -1,17 +1,22 @@
 import * as React from 'react'
 import type { DashboardOptions } from '@uppy/dashboard'
-import { Omit, ToUppyProps } from './CommonTypes'
+import { Omit, WithBaseUppyProps } from './CommonTypes'
 
 // This type is mapped into `DashboardProps` below so IntelliSense doesn't display this big mess of nested types
 type DashboardPropsInner = Omit<
-  ToUppyProps<DashboardOptions>,
+  WithBaseUppyProps<DashboardOptions>,
   // Remove the modal-only props
-  'animateOpenClose' | 'browserBackButtonClose' | 'inline' | 'onRequestCloseModal' | 'trigger'
-> & React.BaseHTMLAttributes<HTMLDivElement>
+  | 'animateOpenClose'
+  | 'browserBackButtonClose'
+  | 'inline'
+  | 'onRequestCloseModal'
+  | 'trigger'
+> &
+  React.BaseHTMLAttributes<HTMLDivElement>;
```

Comment:
Is the linter misconfigured?
```suggestion
  React.BaseHTMLAttributes<HTMLDivElement>
```